processor/processortest/shutdown_verifier.go

```diff
@@ -44,9 +44,61 @@ func verifyTracesDoesNotProduceAfterShutdown(t *testing.T, factory processor.Fac
 	assert.EqualValues(t, generatedCount, nextSink.SpanCount())
 }
 
+func verifyLogsDoesNotProduceAfterShutdown(t *testing.T, factory processor.Factory, cfg component.Config) {
+	// Create a proc and output its produce to a sink.
+	nextSink := new(consumertest.LogsSink)
+	proc, err := factory.CreateLogsProcessor(context.Background(), NewNopCreateSettings(), cfg, nextSink)
+	if err != nil {
+		if errors.Is(err, component.ErrDataTypeIsNotSupported) {
+			return
+		}
+		require.NoError(t, err)
+	}
```

Comment:
```suggestion
	if errors.Is(err, component.ErrDataTypeIsNotSupported) {
		return
	}
	require.NoError(t, err)
```

Reply:
@atoulme when I comment somewhere in your PR about a refactoring like this, I expect you to apply it everywhere else. Please, also you should do it for trace.

Reply:
sure thing!